R/Lithuania.R

```diff
@@ -401,13 +401,12 @@ Lithuania <- R6::R6Class("Lithuania",
     #'   returned?  (Defaults `FALSE`)
     #' @param all_osp_fields A logical scalar. Should all the meaningful
     #'   data fields from the OSP source be returned? (Defaults `FALSE`)
-    #'
-    #' @param ... Parameters passed to `initialise_dataclass`.
+    #' @param ... Parameters passed to `DataClass` initalize
```

Comment:
you should be able to use a roxygen package linking to maybe force this: https://stackoverflow.com/questions/25489042/linking-to-other-packages-in-documentation-in-roxygen2-in-r

or it might just need to be `DataClass()`